Azure/connector_azure_flow_logs_collector.json

```
@@ -0,0 +1,53 @@
+{
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Connector Azure Flow Logs configuration",
+    "type": "object",
+    "properties": {
+      "container_name": {
+        "description": "Container name to interact with Azure Flow Logs",
+        "type": "string"
+      },
+      "account_name": {
+        "description": "Account name of the Azure Flow Logs",
+        "type": "string"
+      },
+      "account_key": {
+        "description": "Account key of the Azure Flow Logs",
+        "type": "string"
+      },
+      "intake_server": {
+        "description": "Server of the intake server (e.g. 'https://intake.sekoia.io')",
+        "default": "https://intake.sekoia.io",
+        "type": "string"
+      },
+      "intake_key": {
+        "description": "Intake key to use when sending events",
+        "type": "string"
+      },
+      "chunk_size": {
+        "type": "integer",
+        "description": "The max size of chunks for the batch processing",
+        "default": 1000
+      },
+      "frequency": {
+        "type": "integer",
+        "description": "Batch frequency in seconds",
+        "default": 60
+      }
+    },
+    "required": [
+      "container_name",
+      "account_name",
+      "account_key",
+      "intake_key"
+    ],
+    "secrets": [
+      "account_key"
+    ]
+  },
+  "description": "connector playbook to get Azure Flow Logs events information",
+  "docker_parameters": "azure_flow_logs",
+  "name": "Collect Azure Flow Logs events",
```

I suggest using the full name in order to separate from the other integration
```suggestion
  "name": "Collect Azure Network Watcher VirtualNet Flow Logs",
```
I also suggest renaming this other integration:
```
Collect Azure Network Watcher Network Security Group Flow logs
```